### PR TITLE
🧹 Add Klaytn, Xchain, and Rarible api urls

### PR DESCRIPTION
**Pull request by ravinagill15**
## In this PR:
- Added api urls for klaytn, xchain, and rarible
- Fixed small typo

### CODE REVIEW

**Comment by TRileySchwarz** [comment]
changeset~

**Comment by ravinagill15** [comment]
Setting this as `do not merge` because adding the `--api-url` arg to the contract verification tool, eg. `npx @layerzerolabs/verify-contract --network xchain-mainnet --api-url "https://xchain-explorer.idex.io/api"` essentially does the same thing as this PR.

Also because verifying contracts on klaytn using `npx @layerzerolabs/verify-contract --network klaytn-mainnet --api-url "https://api-baobab.klaytnscope.com/api"` **does not work at all and requires further investigation **

Verifying contracts on xchain and rarible using the api urls in this PR with the `--api-url` arg to the contract verification tool **works, but does not fix the partial match on FiatTokenV2_2.sol**

File: packages/verify-contract/src/common/url.ts [hunk 1]
```diff
@@ -77,6 +77,8 @@ const DEFAULT_SCAN_API_URLS: Map<NetworkName, string> = new Map([
     ['kava', 'https://kavascan.com/api'],
     ['kava-mainnet', 'https://kavascan.com/api'],
     ['kava-testnet', 'https://testnet.kavascan.com/api'],
+    ['klaytn', 'https://api-baobab.klaytnscope.com/api'],
+    ['klaytn-mainnet', 'https://api-baobab.klaytnscope.com/api'],
```
Comment (ravinagill15) [inline packages/verify-contract/src/common/url.ts:81]: this may be testnet...looking into this

Comment (ravinagill15) [reply packages/verify-contract/src/common/url.ts:81]: yup -- mainnet api url is "https://api-cypress.klaytnscope.com/api", will update here soon

**Review by TRileySchwarz — APPROVED** [review]

**Pull request merged**